src/open_r1/configs.py

```diff
@@ -32,8 +32,10 @@ class GRPOConfig(trl.GRPOConfig):
     callbacks: list[str] = field(
         default_factory=lambda: [], metadata={"help": "The callbacks to run during training."}
     )
+    chat_template: Optional[str] = field(default=None, metadata={"help": "The chat template to use."})
     system_prompt: Optional[str] = field(
-        default=None, metadata={"help": "The optional system prompt to use for benchmarking."}
+        default="You are a helpful AI Assistant, designed to provided well-reasoned and detailed responses. You FIRST think about the reasoning process as an internal monologue and then provide the user with the answer. The reasoning process MUST BE enclosed within <think> and </think> tags.",
```

Comment:
Note that this default differs from the hard-coded system prompt we had previously to emit the `<answer>` and `</answer>` tags. I found it worked pretty nicely